MU-1573 - Sleep Processor (#74)
mulesoft/docs-munit: modules/ROOT/pages/sleep-processor.adoc

> = Sleep Processor

Reviewer: I did an online search, and I can't find any references at all to a sleep processor. I have seen processor sleep mode, but that is about it. Is this something new? Is it a product, feature, component, service? Does it "belong" to Mule?

I guess if I have to ask all these questions, maybe we need a longer introduction?

"A _sleep processor_ is part of <what?>. It is a <blah blah> that enables you to pause...."

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

The Sleep processor allows you to pause your test for a specific time. You can use this processor to simulate a delay in the processing of the flow:

[source,xml,linenums]
----
<munit-tools:sleep time="100" timeUnit="MILLISECONDS"/>
----

> 

Reviewer: Do we really need a table for just two items that are terms + definitions?
Suggest using definition list; see style ref.

[%header%autowidth.spread]
|===
|Attribute Name |Description

|`time`

> | The time period your test will wait for sleep.

Reviewer: ```suggestion
| Numerical value for the unit of time that your test waits before going to sleep
```

|`timeUnit`

> | Specifies the unit in which the `time` value is measured.

Reviewer: ```suggestion
| The unit in which the `time` value is measured
```

Possible values are:

* `MICROSECONDS`

> * `MILLISECONDS`

Reviewer: ```suggestion
* `MILLISECONDS` (default)
```

* `MINUTES`
* `NANOSECONDS`
* `SECONDS`
* `HOURS`

> If not specified, the default value is `MILLISECONDS`.

Reviewer: OMIT if accepting suggestion above.
